source/operator.rst

```diff
@@ -2660,7 +2647,8 @@ Percona Server for MongoDB backups.
 +-----------------+---------------------------------------------------------------------------------------------+
 |                 | .. _backup-storages-verifytls:								|
 |                 |												|
-| **Key**         | `backup.storages.<storage-name>.verifyTLS <operator.html#backup-storages-verifytls>`_	|
+| **Key**         | `backup.storages.<storage-name>.insecureSkipTLSVerify <operator.html#			|
```

Comment:
`insecureSkipTLSVerify` should go under `S3`, so it would be like:
```
backup.storages.<storage-name>.s3.insecureSkipTLSVerify
```
@hors please just double-check for me.